manifest/v1alpha/slo/slo.go

```diff
@@ -68,7 +68,7 @@ type Attachment struct {
 // ObjectiveBase base structure representing an objective.
 type ObjectiveBase struct {
 	DisplayName string   `json:"displayName"`
-	Value       *float64 `json:"value"`
+	Value       *float64 `json:"value,omitempty"`
```

Comment:
please add `// Deprecated:` comment as per https://go.dev/wiki/Deprecated

Reply:
I can't add it here because it is only deprecated for Composite SLOs. For other types of SLOs (threshold and ratio) it is still required.

Reply:
huh, didn't catch it, thanks